src/components/RenderList.vue

```diff
@@ -77,17 +78,21 @@ export default {
       handleInput(event) {
         this.value = event.target.value;
       },
-      addIngredient() {
+      addIngredient(index) {
+        const newID = uuid.v4();
         const newIngredient = {
-          id:Date.now(),
+          id:newID,
```

Comment:
in js object the spacing shoul be: `name: value`

Reply:
✔️